addons/certificate/models/key.py

```diff
@@ -250,7 +317,14 @@ def _numbers_public_key_bytes_with_key(self, pem_key, formatting='encodebytes'):
 
     @api.model
     def _generate_ec_private_key(self, company, name='id_ec', curve='SECP256R1'):
-
+        ''' Generate an elliptic curve private key
+
+        :param res.company() company:   A company record
```

Comment:
Since it's the type, it's probably better without the parenthesis.
```suggestion
        :param res.company company:   A company record
```